frontend/model/notifications/templates.js

```diff
@@ -241,10 +307,18 @@ export default ({
       name: !isCreator ? `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${creatorID}` : ''
     }
 
+    const plaintextArgs = {
+      ...options,
+      ...LTags('strong'),
```

Comment:
I thought plaintext didn't have markup?

Reply:
Right, that shouldn't be there. Good catch. Also, I think I'll replace the @ with the actual display name